perfkitbenchmarker/linux_benchmarks/mysql_service_benchmark.py

```diff
@@ -840,9 +863,26 @@ def Prepare(benchmark_spec):
   # Setup common test tools required on the client VM
   vms[0].Install('sysbench05plus')
 
+
+  vms[0].mysql_svc_oltp_tables_count = FLAGS.mysql_svc_oltp_tables_count
```

Comment:
chatted offline, we can simply change these two functions to pass in the benchmark_spec or the metadata object which also captures required information from benchmark_spec.